docs/how-to-guides/software-management/isis-release-schedule.md

```diff
@@ -0,0 +1,44 @@
+# ISIS Release Schedule
+This document describes the cadence and schedule for ISIS releases.
+
+## Release Cadence
+Releases and development of ISIS3 follows a time based schedule with a new release occurring every three months. Below, we illustrate a sample four month snapshot of software development.
+
+<figure markdown>
+  ![Release Cadence](../../../assets/release-schedule/release_schedule.png)
+  <figcaption>Example of ISIS Release Cadence</figcaption>
+</figure>
+
+
+
+At the start of Month 1, a Release Candidate (RC1) is created from the `dev` branch of our GitHub repository. This RC contains all development from the previous (not shown) three months. RC1 is made publicly available as both a labelled branch and via our Anaconda.org (conda) download page. During Month 1, we solicit input and testing from the broader community. Any issues identified in RC1 will be fixed during Month 1. At the conclusion of Month 1, the release is packaged and the next ISIS3 release is made available for the general public using Anaconda.org (and the default `main` label).
```

Comment:
Do you think linking ISIS's Anaconda page may be helpful too? https://anaconda.org/usgs-astrogeology/isis
I see it being mentioned again in the **Release** section, so maybe we can either hyperlink it there too or we could add a separate link somewhere on this page pointing to ISIS's Anaconda page?